docs/tutorials/how_to/span_representations.md

```diff
@@ -110,54 +110,6 @@ assert spans == [(0, 1), (0, 2), (1, 2), (1, 3), (2, 3)]
 There are other helpful functions in `allennlp.data.dataset_readers.dataset_utils.span_utils`,
 such as a function to convert between BIO labelings and span-based representations.
 
-### Use a Pruner
```

Comment:
I removed this section in the tutorial because it doesn't feel that closely related to span representations, especially after this change. But please lmk if we want to keep it but change it to use the new `masked_topk`.